### PR TITLE
Pathweaver

### PR DESCRIPTION
**Pull request by EliyaFishman**
I checked using the simulator and apparently generating trajectories is faster than loading them from files. This is the reason I removed saving generated trajectories from the code.
This PR only adds loading trajectories from pathweaver files.

### CODE REVIEW

**Comment by EliyaFishman** [comment]
Wait for the folders PR to be merged before merging this.

File: src/main/java/frc/robot/motion_profiling/Path.java
Comment (HilelAmiel) [inline src/main/java/frc/robot/motion_profiling/Path.java]: You know what, it makes me wonder. would we even need to get or set the startVelocity or endVelocity? if the answer is no, then remove it because it is unnecessary.

File: src/main/java/frc/robot/motion_profiling/Path.java
Comment (HilelAmiel) [inline src/main/java/frc/robot/motion_profiling/Path.java:63]: good.

File: src/main/java/frc/robot/motion_profiling/Path.java
Comment (HilelAmiel) [inline src/main/java/frc/robot/motion_profiling/Path.java]: trajectory =  new Trajectory(List.of(new Trajectory.State()));
too many spaces:
trajectory =  new 
To
trajectory = new

Comment (HilelAmiel) [inline src/main/java/frc/robot/motion_profiling/Path.java]: + " initializing with empty path instead of loading");
TO
                    + " initializing with an empty path");

File: src/main/java/frc/robot/motion_profiling/Path.java
Comment (HilelAmiel) [inline src/main/java/frc/robot/motion_profiling/Path.java]: System.err.println("could not load path from: " + path.toString()
Big C pls and print the path name first:
            System.err.println("Could not load " + pathName + " path from: " + path.toString()

File: src/main/java/frc/robot/enums/AutoPath.java
Comment (HilelAmiel) [inline src/main/java/frc/robot/enums/AutoPath.java]: Very well, add example path you created in the GUI and check with the simulation that its work (generate the path without error or other things)

**Review by HilelAmiel — CHANGES_REQUESTED** [review]
little fix and its done

**Pull request merged**
